### ISSUE TITLE
AXGrid listLoadingDisplay() 직후 결과출력전 contentScroll, resize 

### ISSUE THREAD

**Issue by aaboo** [axisj/axisj#725]
Step1. AXGrid 로 조회되어 결과리스트가 출력된 상태에서, 
Step2. 조회를 재시도 하면서 아래처럼 로딩을 걸고
```javascript
myGrid.target.listLoadingDisplay();
```
Step3. 결과가 출력되기전에 윈도우를 resize하거나, contentScroll 을 하게 되면
Step4. 이전 결과물이 나타나고 스크롤링 됩니다.

   > 그럼 마치 이전 출력결과물이 새로 조회된 결과물처럼 보이게 되죠.
   > 잠시 후, 새로운 결과물로 바뀌어 지기까지 하네요

살려주세요

**Comment by lhslive** [comment]
setList나 
reloadList를 통해서 조회를 안하시는건가요?

listLoadingDisplay를 따로 거시는 이유가 궁금하네요

**Comment by aaboo** [comment]
요로코롬 되어 있습니다.
결과물리스트가 많은 경우 몇 초간은 멍을 때리시기 때문에 
mask.open() 후에 mask.close() 해도 되겠지만
이게 왠지 그냥 좋아서요
```javascript
this.dom.find("button[id=btnSearch]").click(function(ev){
	ev.preventDefault();	
	var params = {
		a : ds.obj.find("input[id=a]").attr("a")
		, b : ds.obj.find("input[id=b]").val()
		, c : ds.obj.find("select[id=c]").val()
	};
	//console.log(params);
	$.ajax({
		url: "searchEngine.php"
		, type: "post"
		, data: params
		, beforeSend : function(){
			ds.grid.target.listLoadingDisplay(); //로딩 뱅그르르
		}
		, success: function(rs){
			ds.grid.target.setList(rs); //그리드 뿡야
			ds.grid.target.contentScrollResize(true); //스크롤바가 안생기는 경우가 있어서
		}
	});
});```

**Comment by lhslive** [comment]
http://dev.axisj.com/samples/AXGrid/ajax.html 를 보셨겠지만..

저같은 경우에는 setList 할때 ajaxUrl  관련 옵션을 줘서 주로 불러 옵니다.

```
            myGrid.setList({
                ajaxUrl : "loadGrid.php",
                ajaxPars: {
                    "param1": "액시스제이",
                    "param2": "AXU4J"
                },
                onLoad  : function(){
                    //trace(this);
 
		            //myGrid.setFocus(this.list.length - 1);
                }
            });

```

**Comment by aaboo** [comment]
제가 구현한 방식은 myGrid.setList({Array}) 이와 같은 방식이지만
http://jdoc.axisj.com/AXGrid.html#setList 

lhslive 님 처럼 구현해 보니
onDebug 에 onsucc 떠도, 로딩 중이기만하고 결과가 뿌려지질 않는군요. 
onLoad 에 trace(this)도 타질 않구요.

onDebug시 메시지 중 일부 입니다.
> onsucc[object Object],[object Object],[object Object],[object Object]

response 결과물은
content-type : application/json
```javascript
[
{a:1,b:1,c:1},
{a:2,b:2,c:2}
...
]
```
이와 같은 형식이구요.

하지만, onsucc 때 결과물 출력이 안되고, 로딩이미지만 보이더라도
스크롤하거나 윈도우 리사이즈를 하면 여전히 이전 결과물이 나옵니다.

**Comment by aaboo** [comment]
원래 저의 방식에서 setList를 초기화 하니 원하던 문제가 해결되었어요.

```javascript
var ds = this;
this.dom.find("button[id=btnSearch]").click(function(ev){
    ev.preventDefault();    
    var params = {
        a : ds.dom.find("input[id=a]").attr("a")
        , b : ds.dom.find("input[id=b]").val()
        , c : ds.dom.find("select[id=c]").val()
    };
    //console.log(params);
    $.ajax({
        url: "searchEngine.php"
        , type: "post"
        , data: params
        , beforeSend : function(){
            ds.grid.target.setList([]); //추가됨
            ds.grid.target.listLoadingDisplay(); //로딩 뱅그르르
        }
        , success: function(rs){
            ds.grid.target.setList(rs); //그리드 뿡야
            ds.grid.target.contentScrollResize(true); //스크롤바가 안생기는 경우가 있어서
        }
    });
});
```
추가된 한 줄
> ds.grid.target.setList([]);


이젠, 이전 자료가 나오지는 않는군요.
흠.. 그리드는 어려워요
